### PR TITLE
Added showChildRowToggler option

### PR DESCRIPTION
**Pull request by stevenschemers**
Feel free to let me know if you'd prefer a different variable name or implementation.

### CODE REVIEW

File: dist/vue-tables.js [hunk 1]
```diff
@@ -3378,8 +3378,8 @@ module.exports = function (h) {
     perpageValues.push(h(
       "option",
       {
-        attrs: { value: value },
```
Comment (stevenschemers) [inline dist/vue-tables.js]: I'm not 100% why the builder changed lines like this. Let me know if I should build this differently or at all before merging

**Comment by matfish2** [comment]
Thanks! Apologies for the huge delay

**Comment by stevenschemers** [comment]
No worries! Thank you!

**Comment by JamesAskew** [comment]
@stevenschemers is there any way to use this on a row by row basis? I need to show the toggle for some rows, but hide it for others.

**Comment by stevenschemers** [comment]
@JamesAskew Unfortunately not at this stage. Might need to set up an array that can be checked during the creation of each of the rows.

**Comment by JamesAskew** [comment]
@stevenschemers thanks for the confirmation.

**Pull request merged**